acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature

```diff
@@ -14,76 +13,856 @@ Feature: Open ongoing order
+
+    ## 2. Open order
+
+    Given path 'orders/composite-orders', orderId
+    When method GET
+    And header x-okapi-tenant = centralTenant
+    Then status 200
+
+    * def orderResponse = $
+    * set orderResponse.workflowStatus = "Open"
+
+    Given path 'orders/composite-orders', orderId
+    And param deleteHoldings = false
+    And header x-okapi-tenant = centralTenant
+    And request orderResponse
+    When method PUT
+    Then status 204
+
+
+    ## 3. Check instance in poLine after opening order
+    Given path 'orders/order-lines', poLineId
+    And header x-okapi-tenant = centralTenant
+    When method GET
+    Then status 200
+    And match response.instanceId == '#present'
+    * def centralPoLineInstanceId = response.instanceId
+
+
+    ## 4. Verify Instance, Holdings and items in 'centralTenant'
+
+    Given path 'inventory/instances', centralPoLineInstanceId
+    And header x-okapi-tenant = centralTenant
+    When method GET
+    Then status 200
+
+    Given path 'holdings-storage/holdings'
+    And param query = 'instanceId==' + centralPoLineInstanceId
+    And header x-okapi-tenant = centralTenant
+    When method GET
+    Then status 200
+    And match $.totalRecords == 2
+
+    Given path 'inventory/items'
+    And param query = 'purchaseOrderLineIdentifier==' + poLineId
+    And header x-okapi-tenant = centralTenant
+    When method GET
+    And match $.totalRecords == 2
+
+
+    ## Verify Shadow Instance, Holdings and items in 'universityTenant'
+
+    Given path 'inventory/instances', centralPoLineInstanceId
+    And header x-okapi-tenant = universityTenant
+    When method GET
+    Then status 200
+
+    Given path 'holdings-storage/holdings'
+    And param query = 'instanceId==' + centralPoLineInstanceId
+    And header x-okapi-tenant = universityTenant
+    When method GET
+    Then status 200
+    And match $.totalRecords == 2
+
+    Given path 'inventory/items'
+    And header x-okapi-tenant = universityTenant
+    And param query = 'purchaseOrderLineIdentifier==' + poLineId
+    When method GET
+    And match $.totalRecords == 2
+
+
+    ## Close Order
+
+    Given path 'orders/composite-orders', orderId
+    When method GET
+    Then status 200
+
+    * def orderResponse = $
+    * set orderResponse.workflowStatus = "Closed"
+
+    # Update Order to close
+    Given path 'orders/composite-orders', orderId
+    And request orderResponse
+    When method PUT
+    Then status 204
+
+
+    ## Reopen order with deleteHoldings = true
+
+    Given path 'orders/composite-orders', orderId
+    When method GET
+    Then status 200
+    And match $.workflowStatus == "Closed"
+
+    * def orderResponse = $
+    * set orderResponse.workflowStatus = "Open"
+
+    # Update Order to Open
+    Given path 'orders/composite-orders', orderId
+    And param deleteHoldings = true
+    And request orderResponse
+    When method PUT
+    Then status 204
+
+
+    ## Verify Inventory 'Instnace, Holding, Item' existance in member tenant
```

Comment:
small typo, should be centralTenant in comment

Reply:
updated